ChangeLog

```diff
@@ -1,3 +1,8 @@
+2021-05-20  Ilya Zarubin  <ilya.a.zarub@gmail.com>
```

Comment:
Perhaps we should update the date to avoid confusion when looking back at discussions.

Reply:
Done, but I squashed my branch commits and `git push origin +historyelement-class-additions-for-AL ` hence the outdated label (also below)...